spec/controllers/ops_controller/settings/common_spec.rb

```diff
@@ -239,6 +240,50 @@
           expect(assigns(:ntp_servers)).to eq("1.example.com, 2.example.com")
         end
       end
+
+      context 'get advanced config settings' do
+        it 'for selected server' do
+          _guid, miq_server, _zone = EvmSpecHelper.local_guid_miq_server_zone
```

Comment:
To ensure that this code works for any server and not just my_server, this code should

- create a server via Factory.create(:miq_server)
- not use stub_settings.  stub_settings cheats by hacking the ::Settings constant and also makes it looks like every server returns that value.  Since you are testing settings itself, you shouldn't use it here, because it's bypassing the code you want to verify.  Instead create a server object and actually save settings into it, like was done with the parent PR.